traq/accounts/views.py

```diff
@@ -118,15 +118,25 @@ def timesheet(request):
             done_on__lte=interval[1] 
             ).order_by('started_on')
 
-    work_by_date = dict([(d.date(), []) for d in date_list])
+    work_by_date = dict([(d.date(), dict({'work': [], 'in': None, 'out': None, 'total': None})) for d in date_list])
 
     total_hours = timedelta(0)
     
     for w in work:
+        wbd = work_by_date[w.started_on.date()]
```

Comment:
I think w.started_on.date() will return the date object with the year/month/day reflecting the year/month/day in UTC, not the activated timezone (which is what you probably want).

The way to fix it would be `wbd = work_by_date[localtime(w.started_on).date()]`.

(This stuff is fresh in my mind, since I'm dealing with dates and datetimes in the Alma API project).

As an example:

```python
from model_mommy.mommy import make
from traq.tickets.models import Work
import datetime
# notice my work starts on the *first* of January (local time)
w = make(Work, started_on=datetime.datetime(year=2015, month=1, day=1, hour=22))
# fetch the object from the DB
w = Work.objects.get(pk=w.pk)
print(w.started_on.date())
# 2015-01-02 ====> It's the *second* of January
```